packages/@idemsInternational/gdrive-tools/src/commands/download.ts

```diff
@@ -30,7 +30,6 @@ interface IProgramOptions {
   folderId: string;
   fileEntry64?: string;
   outputPath: string;
-  cachePath: string;
```

Comment:
Code tidying to remove redundant custom cache folder (would just be duplicate of content in output folder)